jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2325,8 +2325,8 @@ void java_bytecode_convert_methodt::convert_athrow(
     assume_code.add_source_location() = assume_location;
 
     code_blockt ret_block;
-    ret_block.move_to_operands(assert_code);
-    ret_block.move_to_operands(assume_code);
+    ret_block.move(assert_code);
+    ret_block.move(assume_code);
     c = ret_block;
```

Comment:
`code_blockt ret_block({assert_code, assume_code});` (and other places use `std::move` afterwards).